IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SwitchingDeviceTypeRelay/DocPropertySet.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SwitchingDeviceTypeRelay" UniqueId="89149be1-41bd-0973-4ca5-1c8cf3beaad6" ApplicableType="IfcSwitchingDevice/RELAY" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="NominalHeight_1Fy4XYaJXDTBDDb9f06ofi" />
+		<DocProperty xsi:nil="true" href="Current_1I1maXboTEcP5xShYx_g0" />
```

Comment:
Current or NominalCurrent?

Reply:
Checked again. It requires the actual working current, so Current is more proper. And WorkingVoltage is updated to Voltage https://github.com/bSI-InfraRoom/IFC-Specification/pull/182/commits/e369102bc553b4469958662ea119e1a89443022c

Reply:
Ok, then ok from my side!